README.md

```diff
@@ -1,6 +1,26 @@
 # AI-TTS-Donations
+
 💰 A bot that uses Uberduck AI to make bit donations have an AI voice.
 
+### How to run
+
+1. Clone the repo
+2. Obtain the following keys:
+
+    1. [An Uberduck API key and secret](https://uberduck.ai/account/manage)
+    2. [A Twitch Client Id and secret](https://dev.twitch.tv/console/apps/create)
+        - Add an OAuth Redirect URI of `http://localhost:3000/`
```

Comment:
Not sure if this is the correct port, lmk if it is something different

Reply:
Corrected the port (really weird port lol): 
```suggestion
        - Add an OAuth Redirect URI of `http://localhost:17563/`
```